core/css/decanter.css

```diff
@@ -1892,7 +1892,8 @@ button,
   outline: none;
   text-align: center;
   text-decoration: none;
-  width: 100%; }
+  width: 100%;
+  transition: all .25s ease-in-out; }
```

Comment:
Perhaps we could change 'all' to the specific transition as we have a known state. The `all` performance for the browser isn't as good as a specific transition. See: https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x

Reply:
committed more specific targetting